docs/projects/lottie/layers/rectangle_layer.rst

```diff
@@ -0,0 +1,46 @@
+Rectangle Layer
+===============
+
+Table of Mappings
+-------------------
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       | layers/fill.json -> “o” -> properties/[value.json OR valueKeyframed.json] |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |            layers/shape.json -> “bm” -> helpers/blendMode.json            |
++---------------------+---------------------------------------------------------------------------+
+|        Color        |                      layers/fill.json -> “ty” = “fl”                      |
++---------------------+---------------------------------------------------------------------------+
+|       Point 1       |             Used in calculation of Origin parameter in lottie             |
++---------------------+---------------------------------------------------------------------------+
+|       Point 2       |             Used in calculation of Origin parameter in lottie             |
++---------------------+---------------------------------------------------------------------------+
+|        Expand       |             Used in calculation of Origin parameter in lottie             |
++---------------------+---------------------------------------------------------------------------+
+|        Invert       |                               Not Supported                               |
```

Comment:
I see you added support for invert in the star layer. Is it possible to add here?

Reply:
Actually the invert parameter is not properly supported in star layer. I didn't find any similar property to support invert parameter yet. 
I guess I will remove it from star layer then.